OpenRA.Game/Network/LocalizedMessage.cs

```diff
@@ -62,7 +53,7 @@ static FluentArgumentType GetFluentArgumentType(object value)
 
 	public class LocalizedMessage
 	{
-		public const int ProtocolVersion = 1;
+		public const int ProtocolVersion = 2;
```

Comment:
This isn't actually changing the `LocalizedMessage` protocol, is it?

Reply:
You are right. The fields and contents are the same.